frontend/src/features/properties/components/ParcelFormControls.tsx

```diff
@@ -115,7 +122,7 @@ const ParcelFormControls: React.FunctionComponent<IParcelFormControlsProps> = ({
         )}
       </FormControls>
 
-      {!disabled && (
+      {!disabled && currentTab === ParcelDetailTabs.parcel && keycloak.isAdmin && (
```

Comment:
only show the delete button if the user is an admin and we are on the parcel tab.